app/commands.js

```diff
@@ -53,6 +53,13 @@ const commands = {
       webContents.openDevTools({mode: 'detach'});
     }
   },
+  'editor:copy': focusedWindow => {
+    if (!focusedWindow) {
+      return;
+    }
+    const webContents = focusedWindow.webContents;
+    webContents.copy();
+  },
```

Comment:
Had to add this because `editor:copy` was not being fired with `ctrl+shift+c` on Linux/electron4. It's a bizarre bug since paste and cut still work, I'm opening an issue on the electron repo

Reply:
Can you add in comment what you explain here and the link to the issue?

Reply:
copy/paste still works on macOS